wren-ai-service/pyproject.toml

```diff
@@ -21,6 +21,7 @@ sqlparse = "==0.5.0"
 sqlglot = "==22.5.0"
 orjson = "==3.10.3"
 sf-hamilton = {version = "1.63.0", extras = ["visualization"]}
+aiohttp = "3.9.5"
```

Comment:
should be `aiohttp = "==3.9.5"`?

Reply:
also `sf-hamilton = {version = "==1.63.0", extras = ["visualization"]}`?

Reply:
according to https://python-poetry.org/docs/dependency-specification/#exact-requirements, both of them are the way to specify the exact version for a package. I think we could unify using `==` to specify the version. I'll modify them in the next commit. 👍